pages/support/_community-support.html

```diff
@@ -0,0 +1,11 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <h2>Community Support</h2>
+    <div class="subheader">Included with All Gruntwork Subscriptions</div>
+    <p>
+      All Gruntwork customers receive access to our community-only chat room in Gitter, where you can engage with other
+      Gruntwork customers and get occasional support from Gruntwork engineers.
```

Comment:
```suggestion
      Gruntwork customers and Gruntwork engineers.
```

"occasional support" is a bit wishy-washy. I understand the goal is to make dedicated support seem more valuable, but I don't know that this is the way to do it.

Reply:
Ha, I think I took that language from a sentence you wrote. But sure, I'll update to your language. I'll also clarify that no SLA is provided in this tier.